lib/matplotlib/backends/backend_qt.py

```diff
@@ -261,43 +261,49 @@ def enterEvent(self, event):
         # Force querying of the modifiers, as the cached modifier state can
         # have been invalidated while the window was out of focus.
         mods = QtWidgets.QApplication.instance().queryKeyboardModifiers()
+        if self.figure is not None:
+            return
```

Comment:
Isn't this condition backwards (and most of the early returns)?

Reply:
...yes, they are all backwands 🤦🏻